backend/promis/backend_api/views.py

```diff
@@ -217,3 +217,18 @@ def get_queryset(self):
             return get_user_model().objects.filter(username = self.request.user)
         else:
             get_user_model().objects.none()
+            
+class UserProfile(APIView):
+    def get(self, request):
+        if helpers.UserExists(request.user):
+            user = get_user_model().objects.get(username = request.user)
+            return Response({
+                'username' : user.username,
+                'first_name' : user.first_name,
+                'last_name' : user.last_name,
+                'last_login' : user.last_login,
+                'date_joined' : user.date_joined,
+                'email' : user.email
+                })
+        else:
+            return Response('unauthorized')
```

Comment:
Can you please wrap it into object, so logged user is `{logged: true, ..<userdata>..}` and guest is `{logged: false}` ?